personal/jagodaszymczak/reports/zajecia2.Rmd

```diff
@@ -0,0 +1,94 @@
+---
+title: "Zajecia2"
+author: "jagodaszymczak"
+date: "4 04 2020"
+output: html_document
+---
+
+```{r setup, include=FALSE}
+knitr::opts_chunk$set(echo = FALSE)
+library(sf)
+library(raster)
+library(spData)
+library(spDataLarge)
+library(dplyr)
+library(ggplot2)
+```
+
+## Zadanie 1
+Wyświetlcie mapę krajów Europejskich  wg liczby ludności
+
+```{r}
+europe <- world[world$continent=='Europe','pop']
+plot(europe)
+```
+---
+
+## Zadanie 2
+
+Który kraj posiada najmniejszą liczbę ludności i ile wynosi?
+
+
+```{r}
+najmniejszy <- world %>% select(name_long, pop) %>%
+  arrange(pop)
+slice(najmniejszy,1)
+```
+---
+
+## Zadanie 3
+
+Ile krajów znajduje się w Azji?
+
+```{r}
+world %>%
+  filter(continent == "Asia")%>%
+  nrow()
+```
+---
+
+## Zadanie 4
+
+Wyświetlcie histogram powierzchni  wszystkich niezależnych krajów (Sovereign Country)
+```{r}
+his <- world %>%
```

Comment:
trochę bym się spierał na ile jest to sensowna nazwa zmiennej, ale tylko trochę ;) .